docs/subnets/customize-a-subnet.md

```diff
@@ -436,6 +437,83 @@ In addition to the AllowList interface, the FeeConfigManager adds the following
 - `getFeeConfigLastChangedAt` - retrieves the timestamp of the last block where the fee config was updated
 - `setFeeConfig` - sets the dynamic fee config on chain (see [here](#fee-config) for details on the fee config parameters)
 
+### Network Upgrades: Enable/Disable Precompiles
```

Comment:
Should this section be the same level as that of Precompiles (with ## instead of ###)? @aaronbuchwald @darioush

Reply:
Since this section refers to precompiles I left it with `###` though since it is referenced at the top we can make it `##` for consistency.

Reply:
Done